Lib/tokenize.py

```diff
@@ -447,7 +447,9 @@ def tokenize(readline):
 
 def _tokenize(rl_gen, encoding):
     source = b"".join(rl_gen).decode(encoding)
-    for token in _generate_tokens_from_c_tokenizer(source, extra_tokens=True):
+    for token in _generate_tokens_from_c_tokenizer(source,
+                                                   extra_tokens=True,
+                                                   ignore_unmatched_parens=True):
```

Comment:
Hummm, maybe we should reuse `extra_tokens` here. Both flags are internal and I don't see us using one instead of the other.